modules/snigelBidAdapter.js

```diff
@@ -104,9 +115,7 @@ export const spec = {
 registerBidder(spec);
 
 function getPage(bidderRequest) {
-  return (
-    getConfig(`${BIDDER_CODE}.page`) || deepAccess(bidderRequest, 'refererInfo.page') || window.location.href
-  );
+  return getConfig(`${BIDDER_CODE}.page`) || deepAccess(bidderRequest, 'refererInfo.page') || window.location.href;
```

Comment:
getConfig(`${BIDDER_CODE}.page`) has no effect correct? if that is set, it should end up in  deepAccess(bidderRequest, 'refererInfo.page')